hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,220 @@
+                  spec:
+                    evaluationInterval:
+                      compliant: 2h
+                      noncompliant: 45s
+                    object-templates:
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: v1
+                          kind: Namespace
+                          metadata:
+                            name: openshift-deployment-validation-operator
+                            labels:
+                              openshift.io/cluster-monitoring: "true"
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: operators.coreos.com/v1alpha1
+                          kind: CatalogSource
+                          metadata:
+                            name: deployment-validation-operator-catalog
+                            namespace: openshift-deployment-validation-operator
+                          spec:
+                            sourceType: grpc
+                            image: ${REGISTRY_IMG}@${IMAGE_DIGEST}
+                            displayName: Deployment Validation Operator
+                            publisher: Red Hat
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: operators.coreos.com/v1
+                          kind: OperatorGroup
+                          metadata:
+                            name: deployment-validation-operator-og
+                            namespace: openshift-deployment-validation-operator
+                            annotations:
+                              olm.operatorframework.io/exclude-global-namespace-resolution: "true"
+                          spec:
+                            upgradeStrategy: TechPreviewUnsafeFailForward
+                            targetNamespaces:
+                              - openshift-deployment-validation-operator
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: operators.coreos.com/v1alpha1
+                          kind: Subscription
+                          metadata:
+                            name: deployment-validation-operator
+                            namespace: openshift-deployment-validation-operator
+                          spec:
+                            channel: ${CHANNEL}
+                            config:
+                              env:
+                              - name: "NAMESPACE_IGNORE_PATTERN"
+                                value: "^(openshift.*|kube-.*|default|dedicated-admin|redhat-.*|acm|addon-dba-operator|codeready-.*|prow)$"
+                            name: deployment-validation-operator
+                            source: deployment-validation-operator-catalog
+                            sourceNamespace: openshift-deployment-validation-operator
```

Comment:
To be changed to the `openshift-marketplace` namespace.

Reply:
@tremes: you can actually keep installing the catalog in the `openshift-deployment-validation-operator` namespace. So please ignore above comment.